src/js/components/cycleCountReporting/IndicatorsTab/IndicatorNumberCards.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+import { RiInformationLine } from 'react-icons/ri';
+import { SortableContainer } from 'react-sortable-hoc';
+
+import LoadingNumbers from 'components/dashboard/LoadingNumbers';
+import NumberCard from 'components/dashboard/NumberCard';
+
+const SortableIndicatorCards = SortableContainer(({ data }) => (
```

Comment:
Let's stick to one component per file.
In other words: let's make the `SortableIndicatorCards` a separate file.